cli2/cli.ts

```diff
@@ -450,20 +465,20 @@ function copyRedistributables(options: CommandOptions, outputPath: string) {
 }
 
 function copyRecursiveSync(src: string, dest: string) {
-  const exists = fs.existsSync(src); 
+  const exists = fs.existsSync(src);
```

Comment:
Exists should be removed here too.

Reply:
fs.exists is deprecated but fs.existsSync.